cornell-dti/office-hours
# New landing page

### Summary 


I implemented the new landing page.

I built the page based on the design for both desktop and mobile devices. I added new components, including a footer, for reuse in other parts of our codebase, as well as changing a lot of the login view component. I still need to add a logo for the footer and add another layer to the curved background, but that should be a quick fix. I will add it to the PR when I get the required assets. 

Some of the things I added based on the design:
- Animations for landing page
- QMI gif was added to the page
- Added curved background
- Changed Google login button design
- Cards giving professors and students an overview of QMI
- New footer




### Test Plan 
- Check to see that the new landing page looks good on both mobile and desktop devices
- Try logging in


### Notes 



### Breaking Changes 

None
